plugins/hydra_ray_launcher/examples/docker/config.yaml

```diff
@@ -0,0 +1,48 @@
+task: 1
+
+hydra:
+  launcher:
+    _target_: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
+    env_setup:
+      pip_packages:
+        omegaconf: null
+        hydra_core: null
+        ray: null
+        cloudpickle: null
+        pickle5: null
+        hydra_ray_launcher: null
+      commands: []
+    ray:
+      cluster:
+        cluster_name: example-cluster
+        min_workers: 0
+        max_workers: 0
+        initial_workers: 0
+        autoscaling_mode: default
+        target_utilization_fraction: 0.8
+        idle_timeout_minutes: 5
+        docker:
+          image: 'samuelstanton/ray-plugin-example'
+          container_name: 'hydra-container'
+          pull_before_run: true
+          run_options: []
+        provider:
+          type: aws
+          region: us-east-2
+          availability_zone: us-east-2a,us-east-2b
+          cache_stopped_nodes: false
+          key_pair:
+            key_name: hydra
+        auth:
+          ssh_user: ubuntu
+        head_node:
+          IamInstanceProfile:
+            Arn: arn:aws:iam::<AWS_ACCOUNT_ID>:instance-profile/ray-head-v1
+          InstanceType: m4.large
+          ImageId: ami-010bc10395b6826fb
+        worker_nodes:
+          IamInstanceProfile:
+            Arn: arn:aws:iam::<AWS_ACCOUNT_ID>:instance-profile/ray-worker-v1
+          InstanceType: m4.large
+          ImageId: ami-010bc10395b6826fb
```

Comment:
Can we use a public AMI image? If not, is it possible to make this one public?

Reply:
This is the `ubuntu-18-04-docker` community AMI